code/stable/gooltest/python/HelloWorld/HelloWorld.py

```diff
@@ -63,6 +63,8 @@
 
 mySlicedList10 = myOtherList[2::z]
 
+mySlicedList10 = myOtherList[2::z]
```

Comment:
Is this output expected? (I'm confused because of the comment ` -- | Do it again, to make sure a unique variable name for endIdx is being generated` above; does that correspond to this line of generated code? Should these lines be the same?)

Reply:
Oh yeah, the reason for that is that Python is the one language that does list slicing correctly 'out of the box', so we don't need to create `endIdx` for it.  All the other renderers do need to create `endIdx` for those examples, though.

Reply:
That's what I get for only checking the Python version cuz it's the easiest 🙃